lib/panoptes/client/authentication.rb

```diff
@@ -22,17 +22,17 @@ def token_expiry
       end
 
       def authenticated?
-        token_contents['id'].present?
+        not token_contents['id'].nil?
```

Comment:
this is more idomatic ruby code
```suggestion
        !!token_contents['id']
```